### ISSUE TITLE
[Feat] Add POST /verify endpoint

### ISSUE THREAD

**Issue by OldMetalmind** [vostpt/api#29]
Add an endpoint that verifies of the user token.

Should expect the `token` field in the body and return:

**200** - Everything is okay
```
{}
```

**400** - Missed fields
```
{
    "token": [
        "This field is required."
    ]
}
```

**401** - Invalid fields
```
{
    "detail": "Token is invalid or expired",
    "code": "token_not_valid"
}
```

**Comment by quetzyg** [comment]
We should probably do a `GET` instead of a `POST`, given we're just asking for a status, but no data is being created. Thoughts?

**Comment by OldMetalmind** [comment]
I most agree with you, I based on an existing example that uses two types of tokens: `access` and `refresh` and this call is also updating the `refresh` token.
For this VOST API makes then more sense to be a `GET`

**Comment by quetzyg** [comment]
What I've got implemented atm is:

**200: OK**
```json
{
  "meta": {
    "expires_in": 2068
  }
}
```

**401: Unauthorised**
```json
{
  "errors": [
    {
      "status": 401,
      "detail": "Token not provided"
    }
  ]
}
```
or

```json
{
  "errors": [
    {
      "status": 401,
      "detail": "Token has expired"
    }
  ]
}
```

**415: Unsupported Media Type**
```json
{
  "errors": [
    {
      "status": 415,
      "detail": "Wrong media type"
    }
  ]
}
```

**Comment by OldMetalmind** [comment]
LGTM 👍

**Comment by quetzyg** [comment]
Fixed via #30
